### PR TITLE
Add docs for affiliated ASPSPs

### PR DESCRIPTION
**Pull request by henrikwidlund**
Document affiliated ASPSP functionality.
Update docs for SWEDSESS.
Update OAS with affiliated ASPSP as well as fix incorrect models for ASPSP Information.
Fix spelling and phrasing.

### CODE REVIEW

File: website/versioned_docs/version-1.0.22/aspsp.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+Read more about the [payment initiation](pis.md) API.
+
+
+## Flows of usage
+
+The APIs in this product does not interface directly with the banks but provide information to be able to access banks dynamically. We will add banks continuously and if you get the list of banks from this API you will get access to these new banks automatically without the need to do any coding. 
+
+A typical flow in the system may be like this:
+1. the user is presented a list of supported countries
+2. the user select a country
```
Comment (martinstenhoff) [inline website/versioned_docs/version-1.0.22/aspsp.md:259]: the user selects a country

File: website/versioned_docs/version-1.0.22/aspsp.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+3. the system retrieves a list of banks for that country
+4. the user selects its bank
```
Comment (martinstenhoff) [inline website/versioned_docs/version-1.0.22/aspsp.md:261]: the user selects their bank

File: docs/aspsp.md [hunk 3]
```diff
@@ -231,7 +224,17 @@ curl -X GET
...
+    "affiliatedAspsps": [
+        {
+            "id": "08999",
+            "name": "Swedbank"
+        },
+        {
+            "id": "08114",
+            "name": "Södra Dalarnas Sparbank AB"
+        }
+    ]
```
[the remark this answers is not in the document]
Comment (henrikwidlund) [reply docs/aspsp.md]: Adjusted, please check.

File: website/versioned_docs/version-1.0.22/aspsp.md (hunk 1 above)
Comment (henrikwidlund) [reply website/versioned_docs/version-1.0.22/aspsp.md]: Adjusted

File: website/versioned_docs/version-1.0.22/aspsp.md (hunk 2 above)
Comment (henrikwidlund) [reply website/versioned_docs/version-1.0.22/aspsp.md]: Adjusted

File: docs/aspsp.md [hunk 4]
```diff
@@ -241,6 +244,8 @@ The list of **global payment products** is generic payments products that we sup
 
 The list of **payment products** is specifically for the bank at hand. When using one of these you have to send payment information in a format the bank will accept. 
 
+The list of **affiliatedAspsps** includes any affiliated ASPSPs to the main entry that can be access when using the `BICFI`. Please see ASPSP specific documentation [here](aspsp.md) for usage.
```
Comment (martinstenhoff) [inline docs/aspsp.md:251]: that can be accessed

**Review by janeklof — APPROVED** [review]

**Review by nikolaykrondev — APPROVED** [review]

File: docs/aspsp.md (hunk 4 above)
Comment (henrikwidlund) [reply docs/aspsp.md]: Adjusted

File: website/versioned_docs/version-1.0.22/aspsp.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+        }
+    ]
+}
+```
+
+This result contains contact details for the bank and information about how to access its services through Open Payments API.
+
+The list of **global payment products** is generic payments products that we support for all banks (where it makes sense). In this case the swedish bank SEB has support for domestic payments internally in Sweden and sepa payments on the European market. When using these we have a unified API for payments that work across banks. If you want a no hassle experience where the API towards us always is the same - this is the products to use.
+
+The list of **payment products** is specifically for the bank at hand. When using one of these you have to send payment information in a format the bank will accept.
```
Comment (martinstenhoff) [inline website/versioned_docs/version-1.0.22/aspsp.md:250]: Isn't this referring to payment raw, which has been removed?

File: website/versioned_docs/version-1.0.22/aspsp_swedsess.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+|![](https://img.shields.io/badge/status-important-important.svg)| Consent, PIS | **Decoupled Only** If PSU has engagement in several banks that belongs to Swedbank, the `X-AffiliatedASPSP-ID` header must be sent. Please see response from [Get ASPSP Details](aspsp.md) for potential values. |
+|![](https://img.shields.io/badge/status-important-important.svg)| AIS, Consent, PIS | In order to use decoupled, please send the `TPP-Redirect-Preferred` header with the value `false` to all calls to AIS, Consent and PIS endpoints. Please do not mix usage of this header, if a consent or payment was created with it specified, all subsequent calls must include it with the same value. This also applies to AIS calls which are referencing consents via the `Consent-ID` header. |
+|![](https://img.shields.io/badge/status-important-important.svg)| AIS | Sandbox: When requesting transactions over 90 days might fail if the same `dateFrom` has been used by any API user recently, resulting in a 401 response with the text `Unauthorized, request not successful. Statement requires SCA`. Please change the parameter value or wait a while before trying again. |
```
Comment (martinstenhoff) [inline website/versioned_docs/version-1.0.22/aspsp_swedsess.md:15]: Sandbox: Requesting transactions over 90 days might fail if the same `dateFrom`  ...

**Review by martinstenhoff — APPROVED** [review]
LGTM

**Pull request merged**